src/VolleyManagement.Backend/VolleyManagement.Services/Infrastructure/IOCServicesModule.cs

```diff
@@ -6,11 +6,11 @@
     using Contracts.Authentication;
     using Contracts.Authentication.Models;
     using Contracts.Authorization;
-    using Contracts.ExternalResources;
     using Crosscutting.Contracts.Infrastructure.IOC;
-    using Mail;
 
+#pragma warning disable S1200 // Classes should not be coupled to too many other classes (Single Responsibility Principle)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)